src/docs/content/controlled.md

```diff
@@ -46,9 +46,16 @@ interaction.
 	import { writable } from 'svelte/store'
 
 	const {
-		elements: { trigger, overlay, content, title, description, close },
-		states: { open }
-	} = createDialog()
+		elements: { 
```

Comment:
I don't think this file needs changing, the structure was correct already and it was more concise 🙂